.classpath

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I see you've added a `.classpath` and `.project` file here. Could you explain what these are for? My idea with TNoodle was that each folder in TNoodle would be a full fledged eclipse project. See https://github.com/jfly/tnoodle/blob/7ed429c25795f830cd2f8333c9609b758f6acaed/README.md#notes for more information. I don't think that would play nice with these top level `.classpath` and `.project` files you've added here.

Reply:
Fixed